tests/security/selinux/audit2allow.pm

```diff
@@ -45,8 +46,9 @@ sub run {
     upload_logs($audit_log_short);
 
     # create an SELinux module, make this policy package active, check the new added module
+    $cmd = "cat " . (is_sle('=15-SP3') ? "$original_audit" : "$audit_log_short") . "| audit2allow -M $test_module";
```

Comment:
How about set a variable for audit log, e.g. `$audit_log = is_sle('=15-SP3') ? "$original_audit" : "$audit_log_short"`, then use this variable in following code.

Reply:
good point, I'll do this :+1: